manifests/service.pp

```diff
@@ -41,6 +65,7 @@
   $id                  = $title,
   $port                = undef,
   $service_name        = $title,
+  $service_config_hash = [],
```

Comment:
Could you add a datatype to the parameter? Also in your example it's a hash, here it is an array.

Reply:
Gotcha, wasn't clear on the datatypes since some had and some don't, and there weren't any in the service manifest.  All should be good now.

Reply:
It takes some time to convert all old code to datatypes, but I think it should be enforced for new parameters.